BRANCHING.md

```diff
@@ -6,18 +6,23 @@ for code reviews. These branches are not the focus of this document, as they
 are generally short lived (as short as a few minutes, or as long as it takes
 for a developer to complete his work).
 
-When working on branch “develop” (equivalent of the improvement branch on Team
+When working on branch “master” (equivalent of the improvement branch on Team
 Foundation Server, our prior source code management system):
 
-1. We will “normally” have one development stream: Branch “develop”. This is
+1. We will normally have one development stream: Branch “master”. This is
 the equivalent of the “improvement” branch in TFS, and represents our
```

Comment:
Well, the intent is that we never break master. We test to be sure our changes are good prior to checkin. If master is broken, we fix it immediately (with priority) as to not impact other developers.

But that said, calling it "stable" without more testing is a stretch. We used to have PBUILD that gave us a lot more confidence across many systems, but that no longer works due to git. We do have plans to bring that back onboard, but we haven't gotten to that yet.

I guess, to summarize: Master "should" work, but if it doesn't, we quickly fix it as soon as we find out about it. After a big merge to Master, things are more likely to be touchy for a few days.